website/content/docs/connect/observability/ui-visualization.mdx

```diff
@@ -95,32 +115,51 @@ to the relevant information.
 
 An example with Grafana is shown below.
 
-<Tabs>
-<Tab heading="HCL">
+<CodeTabs tabs={[ "HCL", "JSON", "YAML (Kubernetes)" ]}>
+
+<CodeBlockConfig>
 
 ```hcl
 ui_config {
   enabled = true
   dashboard_url_templates {
-    service = "https://grafana.example.com/d/lDlaj-NGz/
-      service-overview?orgId=1&var-service={{Service.Name}}&
-      var-namespace={{Service.Namespace}}&
-      var-partition={{Service.Partition}}&var-dc={{Datacenter}}"
+    service = "https://grafana.example.com/d/lDlaj-NGz/service-overview?orgId=1&var-service={{Service.Name}}&var-namespace={{Service.Namespace}}&var-partition={{Service.Partition}}&var-dc={{Datacenter}}"
   }
 }
 ```
 
--> **Note**: the URL is wrapped over multiple lines to make it easier to read
-without horizontal scrolling in the example above however this needs to be a
-normal single-line string value in an HCL configuration file.
+</CodeBlockConfig>
 
-</Tab>
-<Tab heading="Kubernetes YAML">
+<CodeBlockConfig>
 
-On Kubernetes, Consul Server configuration is set in your Helm config via the
-[`server.extraConfig`](/docs/k8s/helm#v-server-extraconfig) key as JSON:
+```json
+{
+  "ui_config": [
+    {
+      "dashboard_url_templates": [
+        {
+          "service": "https://grafana.example.com/d/lDlaj-NGz/service-overview?orgId=1\u0026var-service={{Service.Name}}\u0026var-namespace={{Service.Namespace}}\u0026var-partition={{Service.Partition}}\u0026var-dc={{Datacenter}}"
+        }
+      ],
+      "enabled": true
+    }
+  ]
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig>
 
 ```yaml
+################################### Notes: ####################################
+# On Kubernetes, Consul Server configuration is set in your Helm config via   #
+# the server.extraConfig key as JSON.                                         #
+#                                                                             #
+# The `{{` characters in the URL must be escaped using `{{ "{{" }}` so that   #
+# Helm doesn't try to template them.                                          #
+###############################################################################
+
```

Comment:
@krastin Let's move this information outside of the example. You can use a callout `~> **Note**: ......` to bring attention to this.

Reply:
fixed 👍